index.html

```diff
@@ -3,12 +3,125 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+  <link rel="stylesheet" href="https://use.typekit.net/sbx8kxu.css">
   <link rel="stylesheet" href="style.css">
-  <title>Free thinking Conference: The value of reason, logic, and evidence.</title>
+  <title>Free Thinking Conference: The value of reason, logic, and evidence.</title>
 </head>
-<body>
-  <h1>hello world!</h1>
+<body class="body">
+  <nav class="mobile-menu hide">
+    <img class="mobile-menu-close" src="images/icons/pop-close.svg" alt="close">
+    <ul class="mobile-menu-elements">
```

Comment:
- [OPTIONAL] As suggested by the previous reviewer, kindly consider removing the logo in the mobile version as it is not in the Original design in Behance. instead, you can add `Home` navlink in the mobile menu options to go back to the homepage.
your design
![as](https://user-images.githubusercontent.com/96683843/226126321-dab7ad04-800a-48a4-8ecc-266b16b3d862.jpg)

behance design
![fd](https://user-images.githubusercontent.com/96683843/226126325-67d465ed-50b6-423b-8837-d41ca6362653.jpg)